src/services/component.ts

```diff
@@ -52,10 +52,21 @@ function asyncPropsWrapper<TComponent extends Component>(component: TComponent,
   return defineComponent({
     name: 'AsyncPropsWrapper',
     expose: [],
-    async setup() {
-      const values = await props
+    setup() {
+      const values = ref()
 
-      return () => h(component, values)
+      // eslint-disable-next-line semi-style
+      ;(async () => {
+        values.value = await props
+      })()
```

Comment:
Iffy!

Reply:
self invoking functions brings me back to jquery days lol